irohad/multi_sig_transactions/impl/gossip_propagation_strategy.cpp

```diff
@@ -25,6 +25,7 @@ namespace iroha {
       uint32_t amount)
       : peer_factory(peer_factory),
         non_visited({}),
+        emit_worker(rxcpp::observe_on_new_thread()),
```

Comment:
Now it looks like a dependency of gossip propagation strategy. I think better to explicitly initialize it.